oat-sa/extension-tao-itemqti
# Feat/aut 120/add image caption

[AUT-120](https://oat-sa.atlassian.net/browse/AUT-120)

- https://github.com/oat-sa/tao-item-runner-qti-fe/pull/284

**How to check**
- Create a Block Item and add an image to it
- Click on the image and add some words to the new Image caption textarea on the Widget panel
- Change the Size and Position to verify the caption is working well.
